### PR TITLE
Conversion of Doc without Bib from Mixer to Jekyll.

### PR DESCRIPTION
**Pull request by User141**

### CODE REVIEW

**Comment by User141** [comment]
There is still a TODO in Doc/manuals.html
To solve this, we need the information of the packages which is still in progress.

File: Doc/Examples/rubik.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-<mixertitle> Analyzing Rubik's Cube with GAP </mixertitle>
+---
+title:  Analyzing Rubik's Cube with GAP 
+layout: default
+---
 
 <hr />
 This is an updated GAP&nbsp;4 version of a GAP&nbsp;3
-example by <mixer person="Martin Schoenert" data="name_link"/>, 1993.
+example by {% include namelink.html name="Martin Schoenert" %}, 1993.
```
Comment (AnnaKDS) [inline Doc/Examples/rubik.html]: Should it be Martin Schönert?

File: Doc/Talks/ow97.html [hunk 2]
```diff
@@ -42,39 +42,38 @@
...
-    by <mixer person="Eddie Lo" data="name_link"/> and
-    <mixer person="Gretchen Ostheimer" data="name_link"/><br /><br />
+    by Eddie Lo and
+    Gretchen Ostheimer<br /><br />
   </li>
   <li>
     <a href="newman.tex">newman.tex</a><br />
     <em>Groups with exponent six</em><br />
-    by <mixer person="M. F. Newman" data="name_link"/><br /><br />
+    by {% include namelink.html name="M. F. Newman" %}<mixer person="M. F. Newman" data="name_link"/><br /><br />
```
Comment (AnnaKDS) [inline Doc/Talks/ow97.html]: I forgot to delete the <mixer ../>. Sorry

File: _data/namelink.yml [hunk 3]
```diff
@@ -7,19 +7,18 @@
...
 'Gerhard Hiss': 'http://www.math.rwth-aachen.de/~Gerhard.Hiss/'
 'Götz Pfeiffer': 'http://schmidt.ucg.ie/~goetz'
 'Jean Michel': 'http://www.math.jussieu.fr/~jmichel/'
-'Juergen Mueller': 'http://www.math.rwth-aachen.de/~Juergen.Mueller/'
+'Jürgen Müller': 'http://www.math.rwth-aachen.de/~Juergen.Mueller/'
 'Julianne Rainbolt': 'http://euler.slu.edu/Dept/Faculty/rainbolt/'
 'Klaus Lux': 'http://math.arizona.edu/~klux/'
 'M. F. Newman': 'http://wwwmaths.anu.edu.au/~newman/'
-'Martin Schoenert': 'http://www.math.rwth-aachen.de/~Martin.Schoenert/'
+'Martin Schönert': 'http://www.math.rwth-aachen.de/~Martin.Schoenert/'
```
Comment (fingolfin) [inline _data/namelink.yml]: This is already in `jekyll`, so you may want to rebase your PR on the latest

File: Doc/applications.html [hunk 4]
```diff
@@ -58,7 +56,7 @@
   "http://www.math.rwth-aachen.de/~Juergen.Mueller/preprints/jm104.pdf">
   pdf</a>).
   Lecture Notes,  University of Leicester, MCS Technical Report No. 2004/16,
-  by <mixer person="Juergen Mueller" data="name_link"/>.
+  by {% include namelink.html name="Juergen Mueller" %}.
```
Comment (AnnaKDS) [inline Doc/applications.html]: The test failed. Here should be Jürgen Müller.

File: Doc/applications.html [hunk 5]
```diff
@@ -67,7 +65,7 @@
   <b>An overview on
   <a href="Talks/condensation.html">Condensation&nbsp;Techniques</a></b><br/>
   A summary also pointing to a course of lectures given at the university
-  of Essen in 2003/4 by <mixer person="Juergen Mueller" data="name_link"/>.
+  of Essen in 2003/4 by {% include namelink.html name="Juergen Mueller" %}.
```
Comment (AnnaKDS) [inline Doc/applications.html]: Same here.

Comment (User141) [reply Doc/applications.html]: Done

**Comment by ChrisJefferson** [comment]
Just wanted to check all outstanding changes have been made to this, before I merge it

File: _data/namelink.yml [hunk 6]
```diff
@@ -7,19 +7,18 @@
 'David Joyner': 'https://sites.google.com/site/wdjoyner/'
 'Derek Holt': 'http://www.maths.warwick.ac.uk/~dfh/'
 'Duane Broline': 'http://www.ux1.eiu.edu/~cfdmb/'
-"Eamonn O'Brien": 'http://www.math.auckland.ac.nz/~obrien'
```
Comment (fingolfin) [inline _data/namelink.yml]: Huh, is this on purpose? I think Gap3/Datalib3/small.html still uses it

Comment (AnnaKDS) [reply _data/namelink.yml]: I don't think that we wanted to delete him.

Comment (User141) [reply _data/namelink.yml]: Don't know why this happened.

**Pull request merged**